server/admin/http/handler.go

```diff
@@ -394,3 +404,50 @@ func decodePeerID(id string, w http.ResponseWriter) (peer.ID, bool) {
 	}
 	return peerID, true
 }
+
+func healthCheckValueStore(h *adminHandler) (cause ValueStoreFailReason, err error) {
+	mhash, val, err := createValueStoreDatum()
```

Comment:
You can use static values instead. See comment on `createValueStoreDatum()`

Reply:
done.